### PR TITLE
Upload screenshots to S3 and post comment on Github

### PR DESCRIPTION
**Pull request by janpaul123**
The comment is posted on the commit itself, which should show up in PRs.

The one open question I have is how to deal with “auto merge” in Github.
It could make actual regressions go unnoticed. Any ideas?

### CODE REVIEW

**Comment by stevekrouse** [comment]
Is your thought to disable auto-merge or just recommend that we don't use it?

File: zaplib/scripts/ci/browser_tests.sh [hunk 1]
```diff
@@ -24,12 +24,20 @@ cargo run -p zaplib_ci -- --webdriver-url "https://jpposma_0ZuiXP:${BROWSERSTACK
...
 mv screenshots/ screenshots_report/
 mv previous_screenshots/ screenshots_report/
 mv diff_screenshots/ screenshots_report/
+aws s3 cp --recursive screenshots_report/ s3://zaplib-screenshots/$GITHUB_SHA
+
+if grep --fixed-strings '"newItems":[]' screenshots_report/reg.json && grep --fixed-strings '"deletedItems":[]' screenshots_report/reg.json && grep --fixed-strings '"failedItems":[]' screenshots_report/reg.json
+then
+  echo "SCREENSHOT_GITHUB_MESSAGE=[✅ No screenshot diffs found.](http://zaplib-screenshots.s3-website-us-east-1.amazonaws.com/$GITHUB_SHA)" >> $GITHUB_ENV
+else
+  echo "SCREENSHOT_GITHUB_MESSAGE=[🤔 Screenshot diffs found.](http://zaplib-screenshots.s3-website-us-east-1.amazonaws.com/$GITHUB_SHA) Please look at the screenshots and tag this comment with 👍 or 👎. Only merge when both the PR author and all the reviewers are happy with the changes." >> $GITHUB_ENV
```
Comment (stevekrouse) [inline zaplib/scripts/ci/browser_tests.sh:41]:
```suggestion
  echo "SCREENSHOT_GITHUB_MESSAGE=[🤔 Screenshot diffs found.](http://zaplib-screenshots.s3-website-us-east-1.amazonaws.com/$GITHUB_SHA) Please look at the screenshots and tag this comment with 👍 or 👎. Only merge when both the PR author and a reviewer are 👍." >> $GITHUB_ENV
```

Comment (stevekrouse) [reply zaplib/scripts/ci/browser_tests.sh:41]: ^ to make it clearer that 👍 means that there is no diff

Comment (stevekrouse) [reply zaplib/scripts/ci/browser_tests.sh:41]: And also maybe just require only two 👍 s

Comment (stevekrouse) [reply zaplib/scripts/ci/browser_tests.sh:41]: #nit and up to you if you like it

**Comment by stevekrouse** [comment]
Exciting PR! This is now officially the codebase with the most CI I've ever worked on

File: zaplib/scripts/ci/browser_tests.sh (hunk 1 above)
Comment (janpaul123) [reply zaplib/scripts/ci/browser_tests.sh:41]: > ^ to make it clearer that 👍 means that there is no diff

Oh but there can be a diff! 👍🏻 in that case means that the diff is intentional / desired.

**Comment by stevekrouse** [comment]
I see, so 👍 just means that things are good

**Comment by janpaul123** [comment]
> Is your thought to disable auto-merge or just recommend that we don't use it?

Missed this earlier. It's a good question. I'll disable it for now, and if we feel like that is too annoying then we find a better solution.

**Pull request merged**